### PR TITLE
Added Program 14

### PR DESCRIPTION
**Pull request by tejashrighogare**

### CODE REVIEW

File: perfectNumber/readMe
Comment (RajashreeSonawane) [inline perfectNumber/readMe:1]: add problem statement.

File: perfectNumber/PerfectNumberProgram.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package perfectNumber;
+
```
Comment (jadhavrajashree40) [inline perfectNumber/PerfectNumberProgram.java:2]: Add problem statement

File: perfectNumber/PerfectNumberProgram.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        for (int index = 1; index <= number / 2; index++) {
+            if (number % index == 0) {
+                sumOfNumber = sumOfNumber + index;
+            }
+        }
+        return sumOfNumber;
+    }
+
+    //Write a program for Number is a Perfect Number or not
+    public static void main(String args[]) {
```
Comment (jadhavrajashree40) [inline perfectNumber/PerfectNumberProgram.java:17]: Please add problem statement in proper place.

**Review by jadhavrajashree40 — APPROVED** [review]
Approving PR

File: perfectNumber/PerfectNumberProgram.java (hunk 1 above)
Comment (tejashrighogare) [reply perfectNumber/PerfectNumberProgram.java:2]: Added problem statement

File: perfectNumber/PerfectNumberProgram.java (hunk 2 above)
Comment (tejashrighogare) [reply perfectNumber/PerfectNumberProgram.java:17]: Done

**Review by vrushalee123 — APPROVED** [review]

**Pull request merged**